frontends/web/src/utils/websocket.ts

```diff
@@ -34,7 +39,7 @@ export function apiWebsocket(msgCallback) {
         socket = new WebSocket((isTLS() ? 'wss://' : 'ws://') + 'localhost:' + apiPort + '/api/events');
 
         socket.onopen = function() {
-            socket.send('Authorization: Basic ' + apiToken);
+            socket && socket.send('Authorization: Basic ' + apiToken);
```

Comment:
i prefer `if(socket) { ... }` instead of `&&` and `||` shortcuts. the latter seem a bit hacky to me, as we don't actually care about the result of the (A && B) expression.

Reply:
fixed in https://github.com/digitalbitbox/bitbox-wallet-app/compare/3e9e224a6bdc7f8c435a02e84d945e76e01129c6..79dca499b2c19ed719e18332c610768c5c3adde9